.rubocop.yml

```diff
@@ -4,3 +4,7 @@ inherit_gem:
 
 AllCops:
   TargetRubyVersion: 2.4
+
+Naming/FileName:
+  Exclude:
+    - lib/gds-api-adapters.rb
```

Comment:
Could we add a comment to indicate why this is necessary?

Reply:
```suggestion
    # RuboCop wants this to be renamed to `lib/gds_api_adapters.rb`. But I don't know what that affect that'll have on the rest of this gem and don't particularly have the brain to find out. So I'm ignoring it for now. -- Issy, 2020-05.
    - lib/gds-api-adapters.rb
```